lib/dal/src/audit_logging.rs

```diff
@@ -226,13 +226,12 @@ async fn assemble_for_list(
                         .map_err(Box::new)?
                         .ok_or(AuditLoggingError::ChangeSetNotFound(change_set_id.into()))?;
                     match change_set.status {
-                        ChangeSetStatus::Abandoned
-                        | ChangeSetStatus::Failed
-                        | ChangeSetStatus::Rejected => {
+                        ChangeSetStatus::Failed | ChangeSetStatus::Rejected => {
                             trace!(?change_set.status, ?change_set.id, "skipping change set for audit log assembly due to status");
                             return Ok(None);
                         }
-                        ChangeSetStatus::Applied
+                        ChangeSetStatus::Abandoned
```

Comment:
@stack72 do you want to show all abandoned audit logs? I thought we didn't want to do that.

I suspect this is because you want to show the "I abandoned this change set" event only, but not what you did in that change set before abandoning it. Is that right?

Reply:
yes @britmyerss wanted to show those - believes it's the right level of detail to show